systems/andes_user_guide.rst

```diff
@@ -11,6 +11,27 @@ conduit for large-scale scientific discovery via pre/post processing and
 analysis of simulation data generated on Summit.  Users with accounts on Summit
 will automatically be given access to Andes.
 
+Andes will be available to users soon. All users with current accounts on Rhea will receive account on Andes.
```

Comment:
It might be good to merge this with the previous paragraph to cover both Rhea and Summit user cases

Reply:
Thanks for updating @hliu1300 !